### PR TITLE
DAC-84 + DAC-85

### PR DESCRIPTION
**Pull request by Paroxyse**
Improved error handling
Improved success/error messages
Added retry for endpoints in case of transient failure

### CODE REVIEW

File: DD/wwwroot/Styles/chatWidget.css [hunk 1]
```diff
@@ -93,38 +92,195 @@
     margin-right: 5px;
 }
 
+/* Mejoras para listas en markdown */
```
Comment (TimothyNecr0) [inline DD/wwwroot/Styles/chatWidget.css:95]: Do the comments in english.

File: DD/appsettings.json
Comment (Gapcroz) [inline DD/appsettings.json:1]: "AssistantApiBaseUrl": "http://localhost:5153"

this codeline needs  to be in local???

File: DD/wwwroot/Styles/chatWidget.css [hunk 2]
```diff
@@ -93,38 +92,195 @@
...
+
+/* Input error state */
+.chat-form input.error {
+    border-color: var(--chat-error);
+}
+
+.error-indicator {
+    color: var(--chat-error);
+}
 /* Indicador de carga */
```
Comment (TimothyNecr0) [inline DD/wwwroot/Styles/chatWidget.css:284]: Comments pls

File: DD/wwwroot/Styles/chatWidget.css [hunk 3]
```diff
@@ -93,38 +92,195 @@
...
+.assistant-message ol {
+    padding-left: 40px;
+    margin: 10px 0;
+}
+
+.assistant-message li {
+    margin: 5px 0;
+}
+
 /* Formulario del chat */
```
Comment (TimothyNecr0) [inline DD/wwwroot/Styles/chatWidget.css:106]: Comments

**Review by TimothyNecr0 — CHANGES_REQUESTED** [review]

**Review by Gapcroz — COMMENTED** [review]
in appsettings.json file:

"AssistantApiBaseUrl": "http://localhost:5153/"

this codeline needs to be in local???

**Review by TimothyNecr0 — APPROVED** [review]

File: DD/appsettings.json
Comment (Gapcroz) [reply DD/appsettings.json:1]: I just need to be sure that we are or not using another external url, that's why I'm asking.

**Review by Gapcroz — APPROVED** [review]
approved

**Pull request merged**